modules/luxon.js

```diff
@@ -0,0 +1,10 @@
+/* eslint-disable  */
+import DateTime from '../node_modules/luxon/src/datetime.js';
+
+import {
+  date,
+} from './selectors.js';
+
+const dateTime = DateTime.now().toLocaleString(DateTime.DATETIME_MED);
+
```

Comment:
- [ ] [REQUIRED] The current date is displayed below the navigation bar, for this requirement but current date is not displaying. Please fix this issue!

![image](https://user-images.githubusercontent.com/63915024/192369950-fcfdee97-421c-4d80-94ba-640e2ac52457.png)

Reply:
Hello @mukhammadrizooff. Did you run it on live server? Because it is working for me, time is displayed correctly. Could you inspect if there was an error showing?

Reply:
![image](https://user-images.githubusercontent.com/105621110/192379443-a9b2fd4c-efdb-4311-9faa-3d7cbb3607d9.png)
![Screenshot 2022-09-26 at 18 00 30](https://user-images.githubusercontent.com/105621110/192379703-eb77c160-d608-48b7-a05b-32a94447372f.png)